documentation/docs/recipes/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Recipes
```

Comment:
I'm a bit confused on the differences between Recipes and References. What are your thoughts here?

Reply:
Our references were split fairly cleanly on "how do I do web app'y thing X in Fusion" vs "this is general information about Fusion". So I moved everything in the former into Recipes and everything in the latter into References.

Recipes is my idea of hitting the main points of how to accomplish common web app tasks more than anything else.

Reply:
Ok makes sense